Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>Why adding this one</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Two reasons</p><p> </p><p>We use it extensively in AB Testing and distributions lessons. Adding both for completeness and to ease into this section with something people have already seen</p><p> </p><p>It's also listed in this tweet from Thomas which is where this section derives inspiration from</p><p> </p><p>https://twitter.com/twiecki/status/1419643718110437378</p><p> </p>

Reply:
<p>Let's keep it if you think it's useful</p>